build.gradle

```diff
@@ -58,6 +58,13 @@ allprojects {
     sourceCompatibility = 1.8
     version = version
 
+    eclipse {
```

Comment:
Not the part of PR, but probably correct, right?

Reply:
it's just so IDEs like eclipse or vscode also get the sources and javadocs of the dependencies